src/lib/event_config.c

```diff
@@ -556,3 +556,43 @@ bool check_problem_rating_usability(const event_config_t *cfg,
 
     return result;
 }
+
+GList *expand_event_wildcard(const gchar *event_name, gsize event_len)
```

Comment:
Do we need to pass the length? What’s wrong with good old `strlen()`? Is there ever going to be a non-terminated string passed?

Reply:
Which “GLib XML parser” do you have in mind?

Reply:
Aha, okay, so it’s for the actual parsing, which makes sense.